pkg/controllers/v1alpha2/mpi_job_controller.go

```diff
@@ -1235,7 +1235,7 @@ func (c *MPIJobController) newLauncher(mpiJob *kubeflow.MPIJob, kubectlDeliveryI
 		// issues with scheduler/container technologies.
 		corev1.EnvVar{
 			Name:  "NVIDIA_VISIBLE_DEVICES",
-			Value: "",
+			Value: "none",
```

Comment:
Seems like this is different than “”. See https://github.com/NVIDIA/nvidia-container-runtime/blob/master/README.md. Could you double check?

Reply:
Yes, you are right.